src/types.jl

```diff
@@ -160,6 +160,23 @@ struct RGB4{T<:Fractional} <: AbstractRGB{T}
 end
 RGB4(r::T, g::T, b::T) where {T<:Fractional} = RGB4{T}(r, g, b)
 
+"""`RGB16` is Red-Green-Blue colorspace represented by two bytes (16 bits).
+
+1        |1          |        0
+5 4 3 2 1|0 9 8 7 6 5|4 3 2 1 0 
+---------|-----------|---------
+R R R R R|G G G G G G|B B B B B
+"""
+struct RGB16
```

Comment:
Should be `<: AbstractRGB`

Reply:
I'm getting `LoadError: invalid subtyping in definition of RGB16`

Reply:
Ah, OK, this is a little more complicated. I think you should make it `RGB16 <: AbstractRGB{N2f6}` and then make sure that `red`, `green`, and `blue` all return an `N2f6`. This requires small changes in your mask and shifting.

Alternatively, `RGB16 <: AbstractRGB{Union{N3f5,N2f6}}` *seems* to work, though I wonder if that might cause trouble elsewhere. You could try it and see; for example, if you create an `RGB16` array, in theory ImageView.jl should just display it without complaint. If you encounter errors, this gives you good guidance about what to fix.